charts/alfresco-transform-service/values.yaml

```diff
@@ -304,32 +344,46 @@ filestore:
     data:
       mountPath: "/tmp/Alfresco"
       subPath: "alfresco-content-services/filestore-data"
-activemq:
-  nameOverride: activemq
-  enabled: false
-  # -- Possibility to choose Node for pod, with a key-value pair label
-  # e.g {"kubernetes.io/hostname": multinode-demo-m02}
-  nodeSelector: {}
-  adminUser:
-    # -- Default username for the embedded broker admin user
-    user: admin
-    # -- Default password for the embedded broker admin user
-    password: admin
-# -- external activemq connection setting when activemq.enabled=false
+  volumes: []
+  volumeMounts: []
+
 messageBroker:
+  # -- External activemq connection url (e.g. failover:(nio://my-broker:61616)?timeout=3000&jms.useCompression=true)
   url: null
+  # -- External activemq username
   user: null
+  # -- External activemq password
   password: null
```

Comment:
I don't think we should mention "external" here. All broker are external as managed by the umbrella chart